src/main/java/ch/njol/skript/log/LogEntry.java

```diff
@@ -128,8 +138,25 @@ protected void finalize() {
 	public String toString() {
```

Comment:
I think it's best to keep toString to a more conventional string, maybe make something like toFormattedString for the reformatted output